components/cam/src/physics/cam/micro_p3_utils.F90

```diff
@@ -97,6 +97,8 @@ module micro_p3_utils
 real(rtype), parameter :: incloud_limit = 5.1E-3
 real(rtype), parameter :: precip_limit  = 1.0E-2
 
+logical :: use_cxx = .true.
```

Comment:
I think that micro_p3 and micro_p3_utils should share this value. With the current implementation, there's no way to turn use_cxx off, `p3_init(true);` only impacts the use_cxx in the micro_p3 module. If you make micro_p3 use the one here, things should work.

Reply:
if we setup use_cxx in p3 module, and utils module use it, it will cause circle dependence of module, I would prefer to have use_cxx in utils module, and then p3 use this module, it can be shared and setup.

Reply:
Yes, that's what I meant.